src/components/Badge/tests/Badge.test.tsx

```diff
@@ -3,14 +3,28 @@ import React from 'react';
 import {mountWithAppProvider} from 'test-utilities/legacy';
 import {VisuallyHidden} from 'components';
 
+import {variationName} from '../../../utilities/css';
 import {Badge} from '../Badge';
+import styles from '../Badge.scss';
 
 describe('<Badge />', () => {
   it('renders its children', () => {
     const badge = mountWithAppProvider(<Badge>Badge test</Badge>);
     expect(badge.text()).toBe('Badge test');
   });
 
+  it('accepts a status prop and sets the className for it', () => {
+    const badge = mountWithAppProvider(
+      <Badge status="critical">Badge test</Badge>,
+    );
+    expect(
+      badge
+        .find('span')
+        .first()
+        .hasClass(styles[variationName('status', 'critical')]),
+    ).toBe(true);
+  });
+
```

Comment:
@danrosenthal I am having trouble coming up with an elegant test for `codecov` - let me know if this makes sense

Reply:
Unit tests that check styles are applied aren't very valuable (there's no guarantee that style exists or does any coloring).

Changes in this will be caught by the visual regression tests that run over our examples - and you've now added one of them in storybook.

I'd be tempted to leave this out.

Reply:
leaving it out will fail `codecov` - is that okay in this instance?

Reply:
Yeah I'm fine with that :)

Reply:
done